src/main/kotlin/lotto/Lotto.kt

```diff
@@ -1,23 +1,17 @@
 package lotto
 
-class Lotto(private val number: Set<Int>) {
-    private val _numbers = mutableListOf<Int>()
-    val numbers: List<Int> get() = _numbers
+class Lotto(private val numbers: Set<Int>) {
+
+    val lottoNumber get() = numbers
```

Comment:
```suggestion
class Lotto(val numbers: Set<Int>) {
```
Immutable하게 변경되었으므로, 외부에 노출되더라도 수정될 수 없으므로 별도로 lottoNumber 변수를 만들 필요가 없을 것 같아요